pyproject.toml

```diff
@@ -77,3 +88,13 @@ source = [
 [tool.coverage.report]
 show_missing = true
 precision = 0
+
+[tool.ruff]
+line-length = 100
+target-version = "py39"
```

Comment:
I guess that's the same as `requires-python=">=3.9"` in line 23. Maybe no need to add again ?

Reply:
ah, cool, I didn't know ruff inherited it from pyproject:
`If you're already using a pyproject.toml file, we recommend project.requires-python instead, as it's based on Python packaging standards, and will be respected by other tools. For example, Ruff treats the following as identical to target-version = "py38":`

Thanks!